docs/fml-spec.md

```diff
@@ -0,0 +1,576 @@
+# The Feature Manifest Language
+
+## About this document
+
+This document is the specfiication for the Feature Manifest Language for use with the Nimbus SDK.
+
+Language features that have not yet been implemented are marked as so.
+
+## Introduction
+
+Nimbus is an experimentation platform to allow product owners to easily run experiments on their mobile apps.
```

Comment:
just mobile? :P

Reply:
I'd be very happy if this was expanded to non-mobile platforms!